src/main/java/it/gov/pagopa/pu/workflow/utilities/TaskQueueConstants.java

```diff
@@ -43,4 +43,8 @@ private TaskQueueConstants(){}
 //region Assessments domain
   public static final String TASK_QUEUE_ASSESSMENTS_RESERVED_CREATION = "AssessmentCreationWF";
 //endregion
+
+//region Assessments registry domain
+  public static final String TASK_QUEUE_ASSESSMENTS_REGISTRY_RESERVED_CREATION = "AssessmentsRegistryCreationWF";
```

Comment:
Same domain as Assessments, let's actually configure it as a medium priority WF:
see https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1690697820/Gestione+esecuzione+concorrente+dei+WorkFlow+TaskQueue+e+numero+di+Poller
```suggestion

  public static final String TASK_QUEUE_ASSESSMENTS = "AssessmentsWF";
```